src/command_modules/azure-cli-network/azure/cli/command_modules/network/generated.py

```diff
@@ -260,15 +279,18 @@
 cli_command('network vpn-gateway delete', VirtualNetworkGatewaysOperations.delete, factory)
 cli_command('network vpn-gateway show', VirtualNetworkGatewaysOperations.get, factory)
 cli_command('network vpn-gateway list', VirtualNetworkGatewaysOperations.list, factory)
+cli_command('network vpn-gateway reset', VirtualNetworkGatewaysOperations.reset, factory)
 register_generic_update('network vpn-gateway update', VirtualNetworkGatewaysOperations.get, VirtualNetworkGatewaysOperations.create_or_update, factory)
 
+factory = lambda _: get_mgmt_service_client(VnetGatewayClient).vnet_gateway
+cli_command('network vpn-gateway create', VnetGatewayOperations.create_or_update, factory, transform=DeploymentOutputLongRunningOperation('Starting network vnet-gateway create'))
+
 # VirtualNetworksOperations
 factory = lambda _: _network_client_factory().virtual_networks
 cli_command('network vnet delete', VirtualNetworksOperations.delete, factory)
 cli_command('network vnet show', VirtualNetworksOperations.get, factory)
 cli_command('network vnet list', list_vnet)
-
-cli_command('network vnet update', update_vnet)
+cli_command('network vnet update', update_vnet) # TODO: no tagging
```

Comment:
This is an update command that doesn't support tagging (all updates should support tagging).  I may fix this as part of the IT updates

---
In reply to: [74152169](https://github.com/Azure/azure-cli/pull/668#discussion_r74152169) [](ancestors = 74152169)